Bottom: 6647bd400cd952135fc3715131a0456522f7630c
Top:    9d3797db931c5e93363a725a2ffa088ec342ec89
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-23 19:41:20 +0000

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index a758867..0c4da62 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -298,7 +298,7 @@ draw_right_cross (gint x, gint y)
   glVertex2i (x, 0);
   glVertex2i (x, gport->height);
   glVertex2i (0, y);
-  glVertex2i (gport->width,);
+  glVertex2i (gport->width, y);
 }
 
 static void
